geopmpy/example_agent/example_agent.py

```diff
@@ -0,0 +1,18 @@
+from geopmdpy.session import main
```

Comment:
Not sure this example is more useful than the monitor agent, but might consider adding a better example.